rllib/examples/serving/unity3d_server.py

```diff
@@ -132,6 +133,14 @@ def _input(ioctx):
         .rollouts(
             num_rollout_workers=args.num_workers,
             rollout_fragment_length=20,
+            enable_connectors=False,
+        )
+        .environment(
+            env=None,
+            # TODO: (sven) make these settings unnecessary and get the information
```

Comment:
do you still need to touch this example if it doesn't actually run?

Reply:
ill probably leave this this way because it gets us farther in the debugging process than not leaving this in